src/io/perun/meta.clj

```diff
@@ -1,41 +1,75 @@
 (ns io.perun.meta
   "Utilies for dealing with perun metadata"
-  (:require [boot.core :as boot]))
+  (:require [boot.core :as boot]
+            [clojure.java.io :as io]
+            [clojure.string :as string]
+            [io.perun.core :as perun]))
 
 (def +meta-key+ :io.perun)
 
+(def +global-meta-key+ :io.perun.global)
+
+(def +global-meta-defaults+
+  {:doc-root "public"})
```

Comment:
First, I am not at all sure that this is a good idea (there are probably other ways to change the default values, such as putting them in the global meta file).  

But the basic question is "how does someone change this?" when reading the code, a dynamic var would provide a clear answer.